### PR TITLE
Fix #332, Apply latest contributor license agreement links

### LINKED ISSUE #332: Apply latest contributor license agreement links
**Is your feature request related to a problem? Please describe.**
Changes links from the old contributor license agreement location to its new location.

**Describe the solution you'd like**
Working links

**Additional context**
Depends on https://github.com/nasa/cFS/pull/448

**Requester Info**
Justin Figueroa, ASRC Federal

### PR DESCRIPTION
**Pull request by chillfig**
**Describe the contribution**
- Fixes #332 

Adds checklist for CLA and contributing guide.

**Expected behavior changes**
Working links

**Additional context**
Depends on https://github.com/nasa/cFS/pull/448

**Contributor Info - All information REQUIRED for consideration of pull request**
Justin Figueroa, ASRC Federal

### CODE REVIEW

**Comment by skliper** [comment]
Recommend opening issues in all the affected repos (including PSP), link this PR, and update the commit message to match the recommended format.

**Comment by astrogeco** [comment]
[CCB:2022-04-06](https://github.com/nasa/cFS/wiki/CCB:-2022.04.06) APROVED with CHANGES
- Replace explicit link to CLA files with link to CLA section in cFS/Contributing.md

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -26,9 +31,4 @@ If included, identify any third party code and provide text file of license
 
 **Contributor Info - All information REQUIRED for consideration of pull request**
 Full name and company/organization/center of all contributors ("Personal" if individual work)
-- If NASA Civil Servant Employee or GSFC Contractor on SES II
-  - Address/email/phone and contract/task information (if applicable) must be on file
-- Else if Company
-  - **HAND SIGNED** Company CLA must be on file (once per release): [Company CLA](https://github.com/nasa/cFE/blob/master/docs/GSC_18128_Corp_CLA_form_1219.pdf)
-- Else if Individual
-  - **HAND SIGNED** Individual CLA must be on file (once per release): [Individual CLA](https://github.com/nasa/cFE/blob/master/docs/GSC_18128_Ind_CLA_form_1219.pdf)
+ - Note CLA's apply to software contributions.
```
Comment (astrogeco) [inline .github/pull_request_template.md:34]: no apostrophe `'`, should add the word "only" before software contributions

Comment (skliper) [reply .github/pull_request_template.md:34]: @astrogeco - note that when I tried using the link to the subsection in the cFS contributing guide it didn't go to the appropriate section.  Probably want to confirm this actually works...

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
+**Checklist (Please check before submitting)**
+
+* [ ] I reviewed the [Contributing Guide](https://github.com/nasa/PSP/blob/main/CONTRIBUTING.md).
+* [ ] I signed and emailed the appropriate [Contributor License agreement](https://github.com/nasa/cFS/blob/main/CONTRIBUTING.md#contributor-license-agreement-cla) to GSFC-SoftwareRelease@mail.nasa.gov and copied cfs-program@lists.nasa.gov.
```
Comment (astrogeco) [inline .github/pull_request_template.md:4]: capitalize the `A` in "agreement"

**Review by astrogeco — CHANGES_REQUESTED** [review]

File: .github/pull_request_template.md (hunk 1 above)
Comment (astrogeco) [reply .github/pull_request_template.md:34]: It won't work until nasa/cFS#448 is merged. It does work on the fork https://github.com/chillfig/cFS/blob/add_cla/CONTRIBUTING.md#contributor-license-agreement-cla

**Pull request merged**